spec/supplejack/concept_spec.rb

```diff
@@ -128,5 +128,12 @@ module Supplejack
         end
       end
     end
+
+    describe '#all' do
+      it 'calls the get method' do
```

Comment:
The class.should_receive is the old syntax.
```ruby
SupplejackConcept.should_receive(:get).with('/concepts', {}).and_return({})
SupplejackConcept.all
```

should be:

```ruby
expect(SupplejackConcept).to receive(:get).with('/concepts', {}).and_return({})
```

I don't fully understand what this is really testing and how useful this test is apart from code coverage